### DISCUSSION TITLE
Создание API ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#64] [Announcements]
# Создание API ⚡️ Node.js с примерами кода

Справочник Node.js с примерами кода

https://nodejsdev.ru/guides/metanit/express-api/

**Comment by Arkady0639** [comment]
Не смог найти ? собирал из комметариев к статье. Что-то где-то упустил, поэтому видать users.json  не нашелся, хотя расположен в той же папке что и app.js: webapp. 
А так, статья отличная, многое прояснилось, спасибо.
Так где должен лежать users.json?
